specs/core/core-vocab.ttl

```diff
@@ -520,9 +523,9 @@ oslc:postBody
   rdfs:isDefinedBy oslc: ;
   rdfs:label "postBody" ;
   rdfs:comment """The body of a POST request to return the next page if the response was to a POST request.
- Where a paged resource supports POST with an application/x-www-form-urlencoded body as an alternative to GET to
- avoid the request URI exceeding server limitations, the oslc:ResponseInfo in the response to the POST SHOULD
- contain this property so that a client knows what to POST to get the next page.""" .
+Where a paged resource supports POST with an application/x-www-form-urlencoded body as an alternative to GET to
```

Comment:
I think the whole exercise is futile given that not only whitespace is preserved but also line termination: https://www.w3.org/TR/turtle/#h4_turtle-literals

Just keep it all on a single line or don't bother with all the spacing and expect the reader to clean it up by stripping whitespace from every line and then joining the lines with a space.